velox/experimental/cudf/CudfConfig.h

```diff
@@ -58,6 +60,9 @@ struct CudfConfig {
 
   /// Force replacement of operators. Throws an error if a replacement fails.
   bool forceReplace{false};
+
+  /// Whether to log validation failure messages.
+  bool logValidationFailure{true};
```

Comment:
Should it be true by default?

Reply:
Yes, this information is important, as the spark rapids plugin and Gluten, log the fallback reason is default true